procedures/MT/rotator/rotator_verification.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>Is this a test that is used for formal verification? Meaning it has an LVV-XYZ jira ticket associated with it? </p><p> </p><p>What is the "setup" notebook? Is there a link to it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I am not sure @hdrass has a LVV jira ticket for the Tekniker's verification test or not.

Reply:
Improved the document in commit: [da2822c](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/da2822c02b133ed099461c12c208f882ed94bd6d). Thanks!